server/aws-lsp-codewhisperer/src/shared/codeWhispererService/codeWhispererServiceIAM.ts

```diff
@@ -74,4 +76,85 @@ export class CodeWhispererServiceIAM extends CodeWhispererServiceBase {
             responseContext,
         }
     }
+
+    public async codeModernizerCreateUploadUrl(
+        request: CodeWhispererTokenClient.CreateUploadUrlRequest
```

Comment:
this IAM client should not take the request input of type: CodeWhispererTokenClient.CreateUploadUrlRequest